### PR TITLE
Fixed Bug #5593.

### PR DESCRIPTION
**Pull request by sam09**
Added length validation to location in profile model with maximum length 255. The server does not return an error.

### CODE REVIEW

**Comment by jhass** [comment]
Thanks! Can you add a test exposing the bug?

**Comment by sam09** [comment]
Should I open a separate pull request?

**Comment by jhass** [comment]
No, pushing to the branch you have an pull request open for, updates it :)

**Comment by sam09** [comment]
Ok. Thanks for the help. I am new to the open source community. Hence the confusion.

**Comment by jhass** [comment]
No worries, you're welcome!

**Comment by sam09** [comment]
I can't understand what happened. Why did Travis tests fail

**Comment by jhass** [comment]
Don't worry, we still have a few randomly failing tests unfortunately.

**Comment by sam09** [comment]
Does that mean that the tests I wrote us fine?

**Comment by jhass** [comment]
Yes :)

File: spec/models/profile_spec.rb [hunk 1]
```diff
@@ -112,6 +112,18 @@
...
+  describe "of location" do
+    it "can be 255 characters long" do
+      profile = FactoryGirl.build(:profile, :location => "a"*255)
+      expect(profile).to be_valid
+    end
+   
+    it "cannot be 256 characters" do
+      profile = FactoryGirl.build(:profile, :location => "a"*256)
+      expect(profile).not_to be_valid
+      end
```
Comment (jhass) [inline spec/models/profile_spec.rb]: Please indent the end onto the level of the it ;)

**Comment by jhass** [comment]
Okay, I think it's fair to make this a single commit. Also you committed to develop, which you shouldn't do in the future in case you want to do multiple contributions concurrently ;)  Let's fix that up.

```
git remote add upstream git://github.com/diaspora/diaspora.git
git fetch upstream
git checkout develop
git rebase -i upstream/develop
# replace pick with fix in all except the first line, save & quit
git push -f origin develop
# You can edit the last commit of the current branch with git commit --amend, after which you have to force-push with git push -f
# _After_ this PR is merged, do:
git branch -m develop location_length
git fetch upstream
git checkout -b upstream/develop
git push -f origin develop
# Start your next contributions with something like
git checkout -b my_contribution upstream/develop

**Comment by sam09** [comment]
should i run all of these commands?

**Comment by jhass** [comment]
You should read and maybe try to understand what they do ;) You should especially read the lines that start with `#`, which are comments, some of these will tell you when to run those commands.

**Comment by sam09** [comment]
I think i got it. Thanks for the help

**Comment by jhass** [comment]
Thank you!

**Comment by Flaburgan** [comment]
Thanks!

**Comment by Flaburgan** [comment]
Btw it's nice to see that it doesn't crash anymore, but a message informing the user that the content submitted has been truncated would have be better ;)

**Comment by sam09** [comment]
I would work on that.

**Pull request merged**